### PR TITLE
Akz/feature branch example

### PR DESCRIPTION
**Pull request by geli-gel**
plz review me

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor i
 
 ## Person A:
 
-Person A should modify content here.
+Loream Ispumama
```
Comment (criacuervos) [inline README.md:7]: pretty cool latin

File: favorite-color.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Basic SKY BLUE
```
Comment (criacuervos) [inline favorite-color.txt]: wow nice color

File: scrappy-possums.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+wimble
+scrambler
```
Comment (criacuervos) [inline scrappy-possums.txt:2]: whoahh!!

**Comment by criacuervos** [comment]
this is my comment for the whole review

File: favorite-color.txt (hunk 2 above)
Comment (criacuervos) [reply favorite-color.txt]: can you change this to `doodoo green`

**Review by criacuervos — CHANGES_REQUESTED** [review]

File: favorite-color.txt (hunk 2 above)
Comment (geli-gel) [reply favorite-color.txt]: yes

File: favorite-color.txt [hunk 4]
```diff
@@ -1 +1 @@
-Basic SKY BLUE
+doodoo green
```
Comment (criacuervos) [inline favorite-color.txt:1]: awesome

**Review by criacuervos — APPROVED** [review]
u did great

**Pull request merged**